pkg/controllers/v1alpha2/mpi_job_controller.go

```diff
@@ -923,16 +961,16 @@ func newLauncherRoleBinding(mpiJob *kubeflow.MPIJob) *rbacv1.RoleBinding {
 // newPodGroup creates a new PodGroup for an MPIJob
 // resource. It also sets the appropriate OwnerReferences on the resource so
 // handleObject can discover the MPIJob resource that 'owns' it.
-func newPodGroup(mpiJob *kubeflow.MPIJob, minAvailableReplicas int32) *podgroupv1alpha2.PodGroup {
-	return &podgroupv1alpha2.PodGroup{
+func newPodGroup(mpiJob *kubeflow.MPIJob, minAvailableReplicas int32) *podgroupv1alpha1.PodGroup {
+	return &podgroupv1alpha1.PodGroup{
```

Comment:
Why do we use alpha1 now?

Reply:
Kube-batch still using version alpha1, alpha2 is still in development. when the alpha2 available , I can help bump :).

Reply:
Thanks for your explanation. SGTM